src/javascripts/components/newMushroom/newMushroom.js

```diff
@@ -1,5 +1,9 @@
 import utils from '../../helpers/utils';
 
+function imageInputWatcher() {
+  $('#mush-image-label').html(this.files[0].name);
+}
```

Comment:
this updates the label for the input field so that it shows the image name when you select the file

Reply:
oh also this is different that what we're used to: I wanted to use `this` to get the filename (because that's what various docs told me to do) so I couldn't use an arrow function, I had to use the "old school" function declaration to do so